scripts/entry.js

```diff
@@ -113,14 +113,17 @@ function Entry(data,host)
     html += "</t><t class='link' data-operation='filter:"+r.escape_attr(this.host.json.name)+"-"+this.id+"'>•</t>";
 
     var operation = '';
-    if(this.host.json.name == r.home.portal.json.name)
-      operation = 'edit:'+this.id+' '+this.message.replace(/\'/g,"&apos;");
-    else if(this.whisper)
+    if(this.whisper)
       operation = "whisper:"+this.host.json.name+" ";
     else
       operation = "quote:"+this.host.json.name+"-"+this.id+" ";
 
     html += this.editstamp ? "<c class='editstamp' data-operation='"+r.escape_attr(operation)+"' title='"+this.localtime()+"'>edited "+timeSince(this.editstamp)+" ago</c>" : "<c class='timestamp' data-operation='"+operation+"' title='"+this.localtime()+"'>"+timeSince(this.timestamp)+" ago</c>";
```

Comment:
I don't understand why one has `r.escape_attr` and the other doesn't here.

Reply:
Looks like a modal. it's checking for `this.editstamp`